radix-clis/go.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
What does this do? Is it intended to be committed? Can we add some comments explaining its intention and when it should be run?

Reply:
Added by mistake. Will remove.

Reply:
Done